<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>110</width>
      <height>278</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>posicion en x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="posx_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>posicion en y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="posy_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>masa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mas_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>radio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="redio_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>velocidad en x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="velx_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>velocidad en y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="vely_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CREAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
